crates/gossip/src/initiate.rs

```diff
@@ -576,7 +576,7 @@ mod tests {
             .await;
 
         let mut seen = HashSet::new();
-        for _ in 0..10 {
+        for _ in 0..100 {
```

Comment:
Got a flake on this locally. It's waiting for a random number to select both agents so higher is less likely to flake